packages/atomic/src/utils/slot-utils.ts

```diff
@@ -0,0 +1,23 @@
+export function getSlotFromHost(host: HTMLElement, slotName: string) {
+  const children = Array.from(host.children);
+  const slots = children.filter((child) => {
+    if (slotName) {
+      return child.getAttribute('slot') === slotName;
+    }
+
+    return child.setAttribute('slot', '');
```

Comment:
Why would `slotName` be falsy/empty? I would avoid setting an attribute in a `get*` function.

Reply:
🤔

Reply:
me neither! 🤣